components/SpotDetails.js

```diff
@@ -11,24 +11,24 @@ export default function SpotDetails({ spot }) {
       </DetailsSubList>
       <h3>Wetterempfehlung:</h3>
       <DetailsSubList>
-        {spot.weather.map((weather) => {
+        {spot?.weathersuggestion.map((weather) => {
           return <li>{weather}</li>;
         })}
       </DetailsSubList>
       <h3>Altersempfehlung:</h3>
       <DetailsSubList>
-        {spot.age.map((age) => {
+        {spot?.agesuggestion.map((age) => {
           return <li>{age}</li>;
         })}
       </DetailsSubList>
```

Comment:
Missing key - take a look a the [documentation](https://reactjs.org/docs/lists-and-keys.html)

```suggestion
        {spot?.agesuggestion.map((age) => {
          return <li key={ --> a unique e.g. ID  }  >{age}</li>;
        })}
      </DetailsSubList>
```